Cargo.toml

```diff
@@ -63,7 +63,7 @@ bitflags              = "1.2"
 unicode-width         = "0.1"
 textwrap              = "0.11"
 indexmap              = "1.0"
-os_str_bytes          = "2.2"
+os_str_bytes = { version = "2.3", features = ["raw"] }
```

Comment:
How should this be formatted? No features are used from other crates, so there's nothing similar I can reference.

Reply:
This is fine.